interface/ceed-qfunction.c

```diff
@@ -573,7 +573,8 @@ int CeedQFunctionGetFlopsEstimate(CeedQFunction qf, CeedSize *flops) {
   @param[in]  f          Function pointer to evaluate action at quadrature points.
                            See \ref CeedQFunctionUser.
   @param[in]  source     Absolute path to source of QFunction, "\abs_path\file.h:function_name".
-                           For support across all backends, this source must only contain constructs supported by C99, C++11, and CUDA.
+                           The entire header file must only contain constructs supported by all targeted backends (i.e. CUDA for `/gpu/cuda`,
+                           OpenCL/SYCL for `/gpu/sycl`, etc.). The entire contents of this file and all locally included files are used during JiT compilation for GPU backends.
```

Comment:
```suggestion
                           OpenCL/SYCL for `/gpu/sycl`, etc.). The entire contents of this file and all locally included files are used during JiT compilation for GPU backends and must be available at runtime.
```

Maybe "readable" instead of "available"?

Reply:
Maybe 'All source files must be at the provided firepath at runtime for JiT to function.'

Reply:
Done.